### PR TITLE
post: ng and cli v17.2

### PR DESCRIPTION
**Pull request by cexbrayat**

### CODE REVIEW

File: _posts/2024-02-14-what-is-new-angular-17.2.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+
+The component can then be used like:
+
+{% raw %}
+```html
+<ns-pagination [(page)]="page" [collectionSize]="collectionSize" [pageSize]="pageSize"></ns-pagination>
+```
+{% endraw %}
+
+Note that `page` can be a number or a signal of a number,
```
Comment (jnizet) [inline _posts/2024-02-14-what-is-new-angular-17.2.md:204]: Wow, this looks so wrong to me. I've answered Mathieu Riegler's tweet about this: https://twitter.com/Jean__Meche/status/1755890056261231000.
Angular should let a parent set the value of a signal in a child if it's not an input, IMHO.

If this stays allowed, I would really not encourage it, and would thus drop this note.

File: _posts/2024-02-14-what-is-new-angular-17.2.md [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+This can be enabled by adding a `placeholder` attribute to the directive:
+
+{% raw %}
+```html
+<img ngSrc="logo.jpg" placeholder />
+```
+{% endraw %}
+
+The placeholder is 30px by 30px by default, but you can customize it.
+It is displayed slightly blurred to give a hint to the user that the image is loading. The blur effect can be disabled with `placeholderConfig]="{ blur: false }`.
```
Comment (jnizet) [inline _posts/2024-02-14-what-is-new-angular-17.2.md:261]: Missing opening bracket

File: _posts/2024-02-14-what-is-new-angular-17.2.md [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+
+Another new feature is the ability to use Netlify as a provider,
+joining the existing Cloudflare, Cloudinary, ImageKit, Imgix.
```
Comment (jnizet) [inline _posts/2024-02-14-what-is-new-angular-17.2.md:264]: ImageKit, Imgix -> ImageKit and Imgix.

File: _posts/2024-02-14-what-is-new-angular-17.2.md [hunk 4]
```diff
@@ -0,0 +1,364 @@
...
+```json
+"build": {
+  "builder": "@angular-devkit/build-angular:application",
+  "options": {
+    "clearScreen": true
+  },
+```
+
+You then only see the output of the current build,
+and not from the previous ones.
```
Comment (jnizet) [inline _posts/2024-02-14-what-is-new-angular-17.2.md:319]: ones -> one

**Review by jnizet — CHANGES_REQUESTED** [review]

File: _posts/2024-02-14-what-is-new-angular-17.2.md (hunk 1 above)
Comment (jnizet) [reply _posts/2024-02-14-what-is-new-angular-17.2.md:204]: Wait, I misunderstood. I thought you meant the `page` of the child, but it's the page of the page of the parent. In that case it makse sense. But it should be a WritableSignal, not just a signal. To meke things less confusing, maybe rephrase it as this:

Note that, in the parent component, `page` can be a number, but it can also be a `WritableSignal<number>`. In the latter case, the framework will automatically pass the value of the signal as input to the pagination component, and will set the signal value to the new page when the pagination component emits one.

Comment (cexbrayat) [reply _posts/2024-02-14-what-is-new-angular-17.2.md:204]: Yes exactly, I'll rewrite 👍
